bucket/zeplin.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
+    "license": "MIT",
+    "version": "5.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-5.1.1-full.nupkg#/dl.7z",
+            "hash": "sha1:ada573adf43ce547c815706a1356d739227fef77"
+        },
+        "32bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-5.1.1-full.nupkg#/dl.7z",
+            "hash": "sha1:17a032825fa4d3914c948e47f8377f7dd55451b2"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "bin": "Zeplin.exe",
+    "shortcuts": [
+        [
+            "Zeplin.exe",
+            "Zeplin"
+        ]
+    ],
+    "checkver": {
+        "url": "https://pkg.zeplin.io/windows/latestv2/x64/RELEASES",
+        "regex": "Zeplin-(\\d+.\\d+.\\d+)-full",
+        "reverse": true
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg#/dl.7z"
+            },
+            "32bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-$version-full.nupkg#/dl.7z"
+            }
+        }
```

Comment:
Add hash extraction from `$baseurl/RELEASES`

Reply:
Is there a way to reverse reverse the regex, like there is on `checkver`? I haven't found a way to filter `$baseurl/RELEASES` to only return the latest version and the latest version happens to be at the bottom of the list.

Reply:
`$sha1\\s+$basename`

Reply:
Fixed!